src/xunit.performance.api/Model/AssemblyModel.cs

```diff
@@ -175,6 +175,143 @@ private static IEnumerable<(string testCaseName, string metric, IEnumerable<doub
     }
 
     [Serializable]
+    [XmlRoot("BenchmarkScenario")]
+    public sealed class BenchmarkScenario
+    {
+        [XmlAttribute("Name")]
+        public string Name { get; set; }
+
+        [XmlAttribute("Namespace")]
+        public string Namespace { get; set; }
```

Comment:
I suggested adding support for something like Namespace to allow the scenarios to be grouped when uploading to BenchView, e.g. "RWC.JitBench.MusicStore" instead of just "MusicStore". It doesn't have to be a blocker for Scenario support, but we should implement it soon so that we can more effectively categorize the data into BenchView.

Now that I look at the implementation, though, I wonder whether we can just leverage the implementation from `[Benchmmark]` style perf tests that use the already-implemented `TestModel` class. In this context, `type` is the namespace, `method` is the scenario name, and `name` is the full type+scenario_name+qualifiers (e.g.: type == "RWC.JitBench", method == "MusicStore", name == "RWC.JitBench.MusicStore"). Qualifiers can be things like input data set or scenario workload, etc.

@jorive Any thoughts?